ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectSchemaUtil.java

```diff
@@ -104,6 +107,24 @@ private static void checkMapKeyType(final Schema.Type type) {
     }
   }
 
+  private static Schema convertInt64Schema(final Schema schema) {
+    if (schema.name() == Timestamp.LOGICAL_NAME) {
+      return Timestamp.builder().optional().build();
```

Comment:
It would be good to have a constant defined for this. Perhaps OPTIONAL_TIMESTAMP_SCHEMA? Same question for the convertInt32Schema.

Reply:
Done